tests/service/controllers/test_templates_read_manifest.py

```diff
@@ -17,16 +17,17 @@
 # limitations under the License.
 """Renku service templates read manifest controller tests."""
 import pytest
-from git import GitCommandError
 from marshmallow import ValidationError
 
+from renku.core.errors import GitError
+
 
 def test_template_read_manifest_ctrl(ctrl_init, svc_client_with_templates, mocker):
     """Test template read manifest controller."""
     from renku.service.controllers.templates_read_manifest import TemplatesReadManifestCtrl
 
     cache, user_data = ctrl_init
-    svc_client, headers, template_params = svc_client_with_templates
+    _svc_client, _headers, template_params = svc_client_with_templates
```

Comment:
```suggestion
    _, _, template_params = svc_client_with_templates
```
Since we don't use these values anyways